scipy/optimize/tests/test__root.py

```diff
@@ -3,7 +3,7 @@
 """
 from __future__ import division, print_function, absolute_import
 
-from numpy.testing import assert_
+from numpy.testing import assert_, assert_raises
```

Comment:
`from pytest import raises as assert_raises`

Reply:
I thought it was all handled transparently by a shim somewhere.

Reply:
Not really, numpy's still using nose, and we just did a blanket search-and-replace to avoid shims (back in August 2017 or thereabouts)